scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/DefaultOpEnvironment.java

```diff
@@ -201,7 +231,7 @@ private Set<OpInfo> filterInfos(Set<OpInfo> infos, Hints hints) {
 			// filter out unadaptable ops
 			.filter(info -> !simplifying || !info.declaredHints().contains(
 				Simplification.FORBIDDEN)) //
```

Comment:
AGh, maybe it's time we increase the max code line length from 80 to 120...

Reply:
And/or, you could make the line shorter by doing a static import on the `FORBIDDEN` constant...

Reply:
> And/or, you could make the line shorter by doing a static import on the `FORBIDDEN` constant...

Don't think that's possible - we use both `Simplification.FORBIDDEN` and `Adaptation.FORBIDDEN` - we'd have a clash.

> AGh, maybe it's time we increase the max code line length from 80 to 120...

I don't feel too strongly - I do think there's something to be said about shorter line lengths - maybe some of our API is too verbosely named? **But** now is the time to change it, maybe we should file an issue about a SciJava 3 code style?